rununittest.sh

```diff
@@ -43,6 +47,6 @@ if [ "$RUN_TESTS" == "1" ] ; then
         ${SHARED_OPTIONS} "${TEST_FILTER}" \
         //src:ovms_test > ${TEST_LOG} 2>&1 || \
         test_fail_procedure; } && \
-        tail -n 100 ${TEST_LOG} && \
+        test_success_procedure && \
```

Comment:
Move L51 as well to test_success_procedure. Then we can use this as well in coverage scenario